xmlStyles.go

```diff
@@ -29,6 +29,67 @@ type xlsxStyleSheet struct {
 	ExtLst       *xlsxExtLst       `xml:"extLst"`
 }
 
+// MarshalXML implements xml.Marshaler
```

Comment:
A more thorough comment here explaining why you are copying a bunch of fields to an anonymous struct would help future contributors. I assume this is to get rid of the byte replacements?